storage/types_test.go

```diff
@@ -876,3 +876,19 @@ func BenchmarkFlipRightBit(b *testing.B) {
 		nID.FlipRightBit(27)
 	}
 }
+
+func BenchmarkSplit(b *testing.B) {
+	n := NewNodeIDFromHash(h2b("0000000000000000000000000000000000000000000000000000000000000001"))
+	n.PrefixLenBits = 256
+	for i := 0; i < b.N; i++ {
+		_, _ = n.Split(10, 176)
+	}
+}
+
+func BenchmarkSuffix(b *testing.B) {
+	n := NewNodeIDFromHash(h2b("0000000000000000000000000000000000000000000000000000000000000001"))
+	n.PrefixLenBits = 256
+	for i := 0; i < b.N; i++ {
+		_ = n.Suffix(10, 176)
```

Comment:
For my info, what's special about benchmarking with 10, 176?

Reply:
Nothing, it's just an arbitrary node id